Documenting master_is_stable health API settings (#87901)
elastic/elasticsearch: docs/reference/settings/health-diagnostic-settings.asciidoc

`master_history.max_age`::

> (<<static-cluster-setting,Static>>) The maximum amount of time that the master history is kept in memory

Reviewer: ```suggestion
(<<static-cluster-setting,Static>>) The timeframe we record the master history 
```

> for use in the health diagnostic. Master node changes older than this time will not be considered in the

Reviewer: ```suggestion
to be used for diagnosing the cluster health. Master node changes older than this time will not be considered when
```

> health diagnostic calculations. Defaults to `30m` (30 minutes).

Reviewer: ```suggestion
diagnosing the cluster health. Defaults to `30m` (30 minutes).
```

`health.master_history.identity_changes_threshold`::
(<<static-cluster-setting,Static>>) The number of times that it is _not_ OK to have a master change identity

> to another node. This many changes or more will be reported as a unhealthy. Defaults to `4`.

Reviewer: ```suggestion
Defaults to `4`.
```

`health.master_history.no_master_transitions_threshold`::

> (<<static-cluster-setting,Static>>) The number of times that it is _not_ OK to have a master switch to no

Reviewer: ```suggestion
(<<static-cluster-setting,Static>>) The number of transitions to no master witnessed by a node that indicates the cluster is not healthy.
```
